pkg/apis/core/validation/shoot.go

```diff
@@ -2512,3 +2514,21 @@ func getResourcesForEncryption(apiServerConfig *core.KubeAPIServerConfig) []stri
 
 	return sets.List(resources)
 }
+
+func validateShootManagedIssuer(shoot *core.Shoot) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if helper.HasManagedIssuer(shoot) {
+		kubeAPIServerConfig := shoot.Spec.Kubernetes.KubeAPIServer
+		if kubeAPIServerConfig != nil &&
+			kubeAPIServerConfig.ServiceAccountConfig != nil &&
+			kubeAPIServerConfig.ServiceAccountConfig.Issuer != nil {
+			allErrs = append(allErrs, field.Forbidden(field.NewPath("metadata", "annotations").Key(v1beta1constants.AnnotationAuthenticationIssuer), "managed shoot issuer cannot be enabled when .kuberbetes.kubeAPIServer.serviceAccountConfig.issuer is set"))
+		}
+
+		if helper.IsWorkerless(shoot) {
+			allErrs = append(allErrs, field.Forbidden(field.NewPath("metadata", "annotations").Key(v1beta1constants.AnnotationAuthenticationIssuer), "managed shoot issuer cannot be enabled for workerless shoots"))
+		}
```

Comment:
Why not? 👀

Reply:
See this https://github.com/gardener/gardener/pull/9024#issuecomment-1895787726 for motivation